jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -23,6 +23,7 @@ job('trigger_host_os_build_from_builds_repo') {
       extensions {
```

Comment:
* they are the only ones that use the GitHub pull request build plugin and therefore the only ones that have any context to set
* yes